### PR TITLE
Footer fixed

### PR DESCRIPTION
**Pull request by anand-tiwari**

### CODE REVIEW

File: authentication/logout/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="https://main--eager-nightingale-ec3f88.netlify.app/components/component.css">
+    <link rel="stylesheet" href="../../index.css" />
+    <link rel="stylesheet" href="../../common.css" />
+    <link rel="stylesheet" href="./logout.css" />
+</head>
+
+<body>
+    <div class="container">
+        <nav class="header">
```
Comment (Shivanshsharma01) [inline authentication/logout/index.html:17]: It's good to wrap the main nav in a "header" tag.

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean with good use of variables. Suggesting to use more semantic HTML Tags

**Review by psychoactiv — APPROVED** [review]
Good to merge

File: authentication/logout/index.html (hunk 1 above)
Comment (anand-tiwari) [reply authentication/logout/index.html:17]: resolved

**Pull request merged**